automatic/kitty/tools/chocolateyInstall.ps1

```diff
@@ -1,11 +1,35 @@
-﻿$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+$toolsPath = Split-Path $MyInvocation.MyCommand.Definition
+. $toolsPath\helpers.ps1
+
+StopProcesses
+
+$packageArgs = @{
+    PackageName    = 'kitty'
+    FileFullPath   = Get-Item $toolsPath\kitty-bin-*.zip
```

Comment:
We are trying to move towards removing globbing patterns when it is not needed.
Please update to use a actual path to the zip archive, and have that path being replaced in `update.ps1`.

See the `cmake.install` package as an example:

https://github.com/chocolatey-community/chocolatey-packages/blob/master/automatic/cmake.install/update.ps1#L23

Reply:
Done.